src/applications/mhv-secure-messaging/components/ComposeForm/RecipientsSelect.jsx

```diff
@@ -79,10 +80,13 @@ const RecipientsSelect = ({
 
   return (
     <>
+      <label htmlFor="recipient-dropdown">
+        To <span className="recipient-dropdown-required">(*Required)</span>
+      </label>
+      <CantFindYourTeam />
```

Comment:
I think this `<CanFindYourTeam />` component should be passed as a child to `<VaSelect>` and we can keep `label` attribute. Doing it the way you do here now, introduces an interesting behavior on error state, which I think we may want to avoid

Reply:
And you wont need to introduce that additional css styling for `Required` label